README.md

```diff
@@ -255,25 +255,20 @@ $ make regen
 
 In order to test VM correctness, some vectors exercise situations that should
 not be seen in properly implemented actor code. To induce those situations,
-those vectors rely on two special test harness actors that sit "on the inside"
-and trigger those situations when specific messages are sent to it.
-
- * **Chaos Actor (address `t97`):** exercises behaviours that should be regarded
-   as illegal by the VM. Its ABI spec is part of this testing spec, and it's
-   currently being heavily developed.
-     * Test vectors requiring the **Chaos actor** carry the `chaos_actor:true`
-       selector. 
-     * Refer to the implementation under the [`chaos` package](./chaos).
-     * Once stable, we will document this actor in a spec.
- * **Puppet Actor (address `t98`):** former testing harness that includes
-   methods for invalid sending messages and returning invalid values. It's in
-   the process of being deprecated and consolidated into the **Chaos actor**
-   (track [#61](https://github.com/filecoin-project/test-vectors/issues/61).
-     * Test vectors requiring the **Puppet actor** carry the `puppet_actor:true`
-       selector. 
+those vectors rely on a special test harness actor that sits "on the inside"
+and triggers those situations when specific messages are sent to it.
+
+The **Chaos Actor (address `t97`)** exercises behaviours that should be
```

Comment:
WDYT about replacing this with `t98` (replacing the puppet actor, now that it's out of the picture)?

We also need to update the driver on Lotus side. Could you tackle that?